Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -223,6 +230,13 @@
 
 # Misc
 
+- type: uplinkListing
+  id: UplinkCyberpen
+  category: Misc
+  itemId: CyberPen
+  description: Cybersun's legal department pen. Smells vaguely of hard-light and war profiteering.
+  price: 4
```

Comment:
why is this a separate thing from the other pen, and why does it cost more despite doing less?

Reply:
That's the entire point. It's a mockery of the fact we had an actual honest-to-god pen lying there for this long.

Reply:
Don't see a reason it needs to be a flavor pen then?

Reply:
If anything it's significantly less worthless because it immediately flags someone as having contraband.